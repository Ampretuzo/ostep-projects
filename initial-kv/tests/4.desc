Clear all, put some, get one, delete it and try to get it once again.
