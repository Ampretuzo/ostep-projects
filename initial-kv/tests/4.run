./kv c; ./kv p,one,First p,two,Second p,three,Third; ./kv g,two; ./kv d,two; ./kv g,two
