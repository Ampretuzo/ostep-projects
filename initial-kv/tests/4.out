two,Second
two not found
